claim/README.md

```diff
@@ -83,21 +97,38 @@ can trust the endpoint if necessary.
 Claiming can be performed through a SOCKS5 proxy. To do this when calling the script directly supply the proxy
 endpoint as:
 
-```
-netdata-claim.sh -token=MYTOKEN1234567 -rooms=room1,room2 -proxy=socks5h://127.0.0.1:11081
+```bash
+netdata-claim.sh -proxy=socks5h://127.0.0.1:11081 -token=MYTOKEN1234567 -rooms=room1,room2
 ```
 
-When claiming via the `netdata` binary set the following options in the config:
-```
+When claiming via the `netdata` binary, set the following options in your `netdata.conf` file:
+
+```conf
 [cloud]
     proxy = socks5://X.X.X.X:YYYY
 ```
+
 Proceed to claim using the command-line syntax:
-```
+
+```conf
 /usr/sbin/netdata -D -W "claim -token=MYTOKEN1234567 -rooms=room1,room2"
 ```
 
-Please note - if you supply the proxy endpoint in the configuration then it will also be used to tunnel
-the agent cloud link as well.
+> Please note: if you supply the proxy endpoint in the configuration then it will also be used to tunnel the agent cloud
```

Comment:
```suggestion
> Please note: if you supply the proxy endpoint in the configuration, it is also used to tunnel the agent cloud
```

Reply:
Agent to Cloud connection? Not sure what to understand under "agent cloud"